### PR TITLE
Implement SuperReactions (Burst)

### PR DESCRIPTION
**Pull request by Doc94**
**Description:** This PR handle two things.
- Add support for Create a Color using hex color string (including a few tests)
- Implement SuperReactions and CountDetails in reactions, the methods are called "burst" based in discord docs, not sure if this is a good name or is better in this side of the API call Super...

**Justification:** https://github.com/Discord4J/discord-json/pull/155

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 org.gradle.caching=true
 systemProp.org.gradle.internal.publish.checksums.insecure=true
 version=3.2.7-SNAPSHOT
-discordJsonVersion=1.6.18
+discordJsonVersion=1.6.19-SNAPSHOT
```
Comment (Azn9) [inline gradle.properties:4]:
```suggestion
discordJsonVersion=1.6.18
```

File: core/src/main/java/discord4j/core/object/reaction/ReactionCountDetails.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+     * Gets the data of the reaction.
+     *
+     * @return The data of the reaction.
+     */
+    public ReactionCountDetailsData getData() {
+        return data;
+    }
+
+    /**
+     * Count of super reactions
```
Comment (Azn9) [inline core/src/main/java/discord4j/core/object/reaction/ReactionCountDetails.java:49]: This javadoc and the one for the `getBurst` method are inverted

**Review by Azn9 — CHANGES_REQUESTED** [review]
I think that the public facing methods should be named "super" reactions instead of "burst", as it is how discord names it in their UI.

File: gradle.properties (hunk 1 above)
Comment (Doc94) [reply gradle.properties:4]: If i make that i broke the PR build check, when the PR in json is merged i can edit that here.

**Comment by Doc94** [comment]
> I think that the public facing methods should be named "super" reactions instead of "burst", as it is how discord names it in their UI.

i have that question... if keep burst internally and public call super or use the "discord" mean

**Comment by Azn9** [comment]
> i have that question... if keep burst internally and public call super or use the "discord" mean
This seems the better option to me, so that the internal method uses the same name as the values from the API and in the docs, and as the same time, the users uses the name that is used for end-users

**Review by Azn9 — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (Azn9) [inline gradle.properties:4]:
```suggestion
discordJsonVersion=1.6.19
```

**Pull request merged**